fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -600,21 +601,25 @@ public static int analyzeSchemaVersion(Map<String, String> properties) throws An
         return schemaVersion;
     }
 
-    public static Boolean analyzeUseLightSchemaChange(Map<String, String> properties) throws AnalysisException {
+    public static Boolean analyzeUseFastSchemaEvolution(Map<String, String> properties) throws AnalysisException {
         if (properties == null || properties.isEmpty()) {
-            return Config.allow_default_light_schema_change;
+            return Config.allow_default_fast_schema_evolution;
         }
-        String value = properties.get(PROPERTIES_USE_LIGHT_SCHEMA_CHANGE);
+        String value = properties.get(PROPERTIES_USE_FAST_SCHEMA_EVOLUTION);
         if (null == value) {
-            return Config.allow_default_light_schema_change;
+            value = properties.get(PROPERTIES_USE_LIGHT_SCHEMA_CHANGE);
+            if (null == value) {
+                return Config.allow_default_fast_schema_evolution;
+            }
```

Comment:
```suggestion
            return Config.allow_default_fast_schema_evolution;
```

Reply:
> 

Same with above. Due to the admin test, so i keep the property `light_schema_change` and the above code is for compatibility.